test/fork/LockupDynamic.t.sol

```diff
@@ -396,4 +398,19 @@ abstract contract LockupDynamic_Fork_Test is Fork_Test {
         vars.expectedNFTOwner = params.recipient;
         assertEq(vars.actualNFTOwner, vars.expectedNFTOwner, "post-cancel NFT owner");
     }
+
+    function testFork_LockupDynamic_RebasingAssetConfiguration() external {
```

Comment:
if we are using `vm.warp` isn't the claimable amount going to increase?

shouldn't we also test `claimRebasingAssetYield`?

Reply:
Claimable amount is not a direct function of timestamp but depends on functions [from Blast's SharesBase contract](https://github.com/blast-io/blast/blob/master/blast-optimism/packages/contracts-bedrock/src/L2/Shares.sol).

----------
I also tried writing fork test for [`configureYieldAndGas()`](https://github.com/sablier-labs/v2-core/blob/588dea03ab3382bc1673c150d6e214a12b759b47/src/abstracts/SablierV2Blast.sol#L58C1-L69C6) but it does not work. 

<img width="929" alt="Screenshot 2024-02-27 at 13 31 39" src="https://github.com/sablier-labs/v2-core/assets/6676622/04a2cd49-7d41-421a-b691-96fecbdc6ffb">

The reason is that this function calls an internal function on `YIELD_CONTRACT` which is `0x0000000000000000000000000000000000000100` but it is just a plain address. [Click here](https://testnet.blastscan.io/address/0x0000000000000000000000000000000000000100) for Blastscan.

<img width="716" alt="Screenshot 2024-02-27 at 13 28 07" src="https://github.com/sablier-labs/v2-core/assets/6676622/de2162bf-f886-4a84-bf54-9de06cf1a400"> 

So during fork test, foundry fails to resolve it. However, it works on-chain. You can see all internal txns to yield contract address [here](https://testnet.blastscan.io/address/0x0000000000000000000000000000000000000100/internalTx?amt=0%7E).

Reply:
**i am not** talking about the `Blast` and `Yield` contracts

but the `ERC20Rebasing` [claimable amount](https://github.com/blast-io/blast/blob/78003ff2bd78cb07d3c830a24929d7670b2a1f4d/blast-optimism/packages/contracts-bedrock/src/L2/ERC20Rebasing.sol#L201-L204) (USDB and BWETH)

<img width="362" alt="image" src="https://github.com/sablier-labs/v2-core/assets/99738872/17dcf04c-62ca-4091-9ef7-cab6f3b3dd41">

Reply:
The logic is still the same. It also inherits from `SharesBase`. Are you saying that by increasing timestamp through `vm.warp`, claimable amount should increase? 🤔

Reply:
yeah, mb, i was wrong thinking that `ERC20Rebasing` works differently

we can't test this in foundry until we have the precompiles, right?

Reply:
Yes.

Reply:
Let me try something, I think it can be done.

Reply:
@andreivladbrg check this out: https://github.com/sablier-labs/v2-core/pull/811/commits/ad4ac11fdfc4e987d7e3dab03e40c3d1684d07a8

I have managed to fork test `claimRebasingAssetYield()` for USDB. WETH works differently compared to USDB. It inherits a slightly different version of `ERC20Rebasing` that does not inherit `SharesBase`. So it does not have price variable etc. I looked at its code on-chain but couldn't figure out how they distribute yield in WETH.